lib/connection.js

```diff
@@ -717,6 +717,13 @@ class Connection extends EventEmitter {
   _remoteCallbackWrapper(messageId, error, ...result) {
     this._callback(messageId, error, result);
   }
+
+  _log(message, ...args) {
```

Comment:
It's not actually a `message`, I think, it should be called `eventName`.

Reply:
I agree, but maybe smth like `messageType` will be better.